tenable/io/plugins.py

```diff
@@ -80,15 +80,22 @@ def next(self):
         # If the maptable exists, then graft on the plugin family information
         # on to to the item.
         if self._maptable:
-            fid = self._maptable['plugins'][item['id']]
-            item['family_id'] = fid
-            item['family_name'] = self._maptable['families'][fid]
+            try:
+                fid = self._maptable['plugins'][item['id']]
+                item['family_id'] = fid
+                item['family_name'] = self._maptable['families'][fid]
+            except:
```

Comment:
maybe we should log a warning with the error caught here?
use:
`self._log.warn`

Reply:
The issue is that non-nessus plugins are being passed back in the API.  Things such as OT and NNM plugins.

Reply:
Added warning log to display ids which are not found